frontend/src/web.py

```diff
@@ -72,30 +69,16 @@ def main():
 
     current_dir = os.path.dirname(os.path.abspath(__file__))
 
-    conf = {'global':
-                {
-                    'server.socket_host': '0.0.0.0',
-                    'server.socket_port': args.port or settings.get('frontend', {}).get('port') or 8080
-                },
-            '/':
-                {
-                    'tools.staticdir.root': current_dir
-                },
-            '/static':
-                {
-                    'tools.staticdir.dir': 'static',
-                    'tools.staticdir.on': True
-                },
-            '/manifest.info':
-                {
-                    'tools.staticfile.on': True,
-                    'tools.staticfile.filename': os.path.join(current_dir, '..', 'MANIFEST.MF'),
-                    'tools.auth_basic.on': False
-                }
-
-            }
-
-    tplE.setup(settings)    # setup of global variables and host data for usage in views
+    conf = {
+        'global': {'server.socket_host': '0.0.0.0', 'server.socket_port': args.port or settings.get('frontend',
+                   {}).get('port') or 8080},
+        '/': {'tools.staticdir.root': current_dir},
+        '/static': {'tools.staticdir.dir': 'static', 'tools.staticdir.on': True},
+        '/manifest.info': {'tools.staticfile.on': True, 'tools.staticfile.filename': os.path.join(current_dir, '..',
+                           'MANIFEST.MF'), 'tools.auth_basic.on': False},
+    }
+
+    tplE.setup(settings)  # setup of global variables and host data for usage in views
```

Comment:
Hm, I don't think this re-formatting can be justified.  The original might be imperfect, but the new version doesn't look all that better.  Not to mention it is totally unrelated to the actual change in this branch.